DesignSafe-CI/portal
# Bug/DES-2923: Remove Download Data from My Projects

## Overview: ##
Cite this Data with a citation preview should appear in My Projects. However, Download Citation and the link to the Metrics Modal should not appear in My Projects. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2923](https://tacc-main.atlassian.net/browse/DES-2923)

## Summary of Changes: ##
I'm passing preview to DownloadCitation. If a project is in Preview, the Download Citation does not show. If it is not in Preview, then it does show. 

## Testing Steps: ##
1. In My Projects, verify that Download Citation (Downloads/Views/Citations) and Details (Metrics Modal) do not show
2. In Published, verify that it does show. 

## UI Photos:
My Projects
<img width="615" alt="Screen Shot 2024-06-26 at 11 08 54 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/f71fa32d-5165-44e8-9ffb-9ece973c89a9">

Published

## Notes: ##
